src/app/project/ontology/property-form/property-form.component.ts

```diff
@@ -817,4 +820,32 @@ export class PropertyFormComponent implements OnInit {
         return reason;
     }
 
+    /**
+     * setEditMode: set the mode, i.e. the context in which a property is edited
+     */
+    setEditMode() {
+        if (this.changeCardinalities) {
+            // no editMode - just cards
+            return;
+        }
+        if (this.resClassIri && !!this.propertyInfo.propDef) {
+            // in context of class and with an existing property
+            this.editMode = 'assignExistingProperty';
+        }
+
+        if (this.resClassIri && !this.propertyInfo.propDef) {
+            // in context of class and with an existing property
```

Comment:
this should be "no existing property" right?

Reply:
Indeed a wrong comment.